MapboxNavigation/RouteVoiceController.swift

```diff
@@ -180,24 +167,19 @@ open class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate, AVAudioP
         // Set recentlyAnnouncedRouteStep to the current step
         recentlyAnnouncedRouteStep = routeProgress.currentLegProgress.currentStep
         
-        fallbackText = routeProgress.currentLegProgress.currentStepProgress.currentSpokenInstruction?.text
+        fallbackInstruction = routeProgress.currentLegProgress.currentStepProgress.currentSpokenInstruction
```

Comment:
Remove this line and remove the `fallbackInstruction` property. From now on, it should be the responsibility of `alertLevelDidChange(_:)` to pass `currentSpokenInstruction` directly into `speak(_:)`.